DejaVu.cpp

```diff
@@ -172,7 +172,8 @@ void DejaVu::onLoad()
 	RegisterCVar("cl_dejavu_show_metcount", "Show the met count instead of your record", true, this->showMetCount);
 
 	RegisterCVar("cl_dejavu_visuals", "Enables visuals", true, this->enabledVisuals);
-
+	RegisterCVar("cl_dejavu_toggle_with_scoreboard", "Toggle with scoreboard (instead of always on)", true, this->toggleWithScoreboard);
```

Comment:
What are your thoughts on the default here? You have it set to `true` but I'm guessing that was just a copy/paste artifact.

Reply:
Yes, this was a copy/paste thing. I do think default should be what yours has always been!